src/main/java/freiman/projectile/ProjectileGraph.java

```diff
@@ -0,0 +1,37 @@
+package freiman.projectile;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ProjectileGraph extends JComponent {
+
+    private Projectile projectile = new Projectile(0, 0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+        g.translate(0, getHeight());
+        double seconds = projectile.getSeconds();
```

Comment:
I want you to display the full curve of the projectile, not just up until "seconds"